platform-sdk/swirlds-platform-core/src/test/java/com/swirlds/platform/uptime/UptimeTests.java

```diff
@@ -82,31 +81,31 @@ private static List<EventImpl> generateEvents(
                     .setCreatorId(nodeId)
                     .setConsensusTimestamp(time.now())
                     .build();
-            final EventImpl event = new EventImpl(platformEvent, null, null);
 
             if (!noJudges.contains(nodeId) && firstEventCreated.add(nodeId)) {
-                event.setFamous(true);
-                event.setJudgeTrue();
+                // event.setFamous(true);
```

Comment:
were these commented lines intentionally left here?

Reply:
not intentionally. but there is a PR open that removes this file: https://github.com/hashgraph/hedera-services/pull/14231
to avoid fixing something that is about to be deleted and needing reviews again, I will just leave this as is.